apps/contrib/assets/__tests__/SwitchButton.jest.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { render, screen, fireEvent } from '@testing-library/react'
+import '@testing-library/jest-dom'
+import { SwitchButton } from '../SwitchButton'
+
+test('Show button correctly', () => {
+  render(
+    <SwitchButton
+      switchLabel="This is the label for the switch"
+    />
+  )
+  expect(screen.getByText('This is the label for the switch')).toBeInTheDocument()
+})
+
+test('Switch button should change state onclick', () => {
+  const { getByRole } = render(<SwitchButton />)
+  const switchButton = getByRole('switch')
```

Comment:
I still don't understand why there are two ways to do this but as `screen` is already used above it might be more consistent to do `screen.getByRole` here as well and remove getByRole in line 16